addons/account_edi/models/account_edi_document.py

```diff
@@ -174,6 +174,13 @@ def _postprocess_post_edi_results(documents, edi_result):
                         'error': move_result.get('error', False),
                         'blocking_level': move_result.get('blocking_level', DEFAULT_BLOCKING_LEVEL) if 'error' in move_result else False,
                     })
+                    title = _('An error occurred during posting electronic document')
+                    document.move_id.message_post(
+                        title=title,
+                        body=move_result.get('error', False),
+                        messag_type='comment',
+                        subtype_xmlid='mail.mt_note',
+                    )
```

Comment:
The EDI must not post anything on the chatter. If your CRON needs to try multiple times the same document, you don't want to flood the chatter with that. A good example is the EDI that are only allowed during the day but not accessible during the night.
Btw, I don't understand why you need all this stuff since all errors are available on the invoice EDI header.